services/frontend/pkg/config/defaults/defaultconfig.go

```diff
@@ -184,6 +184,10 @@ func EnsureDefaults(cfg *config.Config) {
 		cfg.MachineAuthAPIKey = cfg.Commons.MachineAuthAPIKey
 	}
 
+	if cfg.HTTP.CORS.AllowedOrigins == nil && cfg.Commons != nil && cfg.Commons.OcisURL != "" ||
+		len(cfg.HTTP.CORS.AllowedOrigins) == 1 && cfg.HTTP.CORS.AllowedOrigins[0] == "*" {
+		cfg.HTTP.CORS.AllowedOrigins = []string{cfg.Commons.OcisURL}
+	}
```

Comment:
@dragonchaser another question. if we run ocis with `OCIS_URL="https://ocis:9200"`, should `cfg.HTTP.CORS.AllowedOrigins` have `["https://ocis:9200"]`

Reply:
What I observed:
1. ocis with `OCIS_URL=https://localhost:9200`
    ```bash
    curl "https://:localhost9200/ocs/v1.php/config" -H"Origin: https://localhost:9200" -vk
    ```
    ```bash
    < Access-Control-Allow-Origin: https://localhost:9200
    < Access-Control-Expose-Headers: Location
    < Content-Length: 266
    ```
2. ocis with `OCIS_URL=https://host.docker.internal:9200` (no cors headers :question:)
    ```bash
    curl "https://host.docker.internal:9200/ocs/v1.php/config" -H"Origin: https://host.docker.internal:9200" -vk
    ```
    ```bash
    < Content-Length: 266
    ```

Reply:
@saw-jan what is needed to do to fix them?

Reply:
>what is needed to do to fix them?

For the API test, I am expecting that when using `OCIS_URL=https://host.docker.internal:9200` the following request should give the cors headers

```bash
curl "https://host.docker.internal:9200/ocs/v1.php/config" -H"Origin: https://host.docker.internal:9200" -vk
```